formatTest/typeCheckedTests/expected_output/jsx.re

```diff
@@ -375,58 +375,62 @@ let thisIsAlsoOkay =
 
 <a /> > <b />;
 
-let listOfListOfJsx = [<> </>];
+let listOfListOfJsx = [ <> </> ];
 
-let listOfListOfJsx = [<> <Foo /> </>];
+let listOfListOfJsx = [ <> <Foo /> </> ];
 
 let listOfListOfJsx = [
   <> <Foo /> </>,
   <> <Bar /> </>
-];
+ ];
 
 let listOfListOfJsx = [
   <> <Foo /> </>,
   <> <Bar /> </>,
   ...listOfListOfJsx
-];
+ ];
```

Comment:
This diff currently uses the `~wrap:("[ ", " ]") config to achieve the spacing, and as a result it causes indentation issues like this.

There's a formatting argument called `~pad:(true, true)` which will ensure there is spaces before the closing and after the opening, which should avoid these issues.